Stresser/FileArtifact.cpp

```diff
@@ -0,0 +1,63 @@
+#include "FileArtifact.h"
+
+FileArtifact::FileArtifact(std::string id, std::string name, std::string type, std::string data) : RuleEntity(id, name, type, data)
+{
+	this->install();
+}
+
+FileArtifact::~FileArtifact()
+{
+	this->unInstall();
+}
+
+void FileArtifact::install() const
+{
+	std::string fileName = this->getData();
+
+	wil::unique_handle hFile(::CreateFileA(
```

Comment:
Use AutoHandle from Utils (static library in the project)
We don't use WIL anymore